Description: relax dependencies
 Depend on env-logger 0.7, which is currently available in Debian.
Author: Emanuele Rocca <ema@debian.org>
Last-Update: 2023-01-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: prost-build/Cargo.toml
===================================================================
--- prost-build.orig/Cargo.toml
+++ prost-build/Cargo.toml
@@ -88,7 +88,7 @@ version = "3"
 version = "4"
 
 [dev-dependencies.env_logger]
-version = "0.8"
+version = "0.7"
 default-features = false
 
 [features]
